docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.4'
+
+services:
+  http:
+    build: .
+    environment:
+      NODE_ENV: development
+    ports:
+      - "${HTTP_PORT:-4001}:4001"
+    volumes:
+      # override packaged /app for dev
+      - .:/app
+      # node_modules may not be platform compatible,
+      # use the container's version stored in the volume instead of the above mount.
```

Comment:
```suggestion
      # use the image's version stored in the volume instead of the above mount.
```

Reply:
The contents of the volume are populated and updated by the container the volume is mounted to, running an instance of the image without the volume mounted only contains the original un-updated copy

Reply:
...you are right. i have a really hard time with this implicit volume copy flow. i wish it was an explicit config option.